ortools/flatzinc/ortools.msc.in

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "com.google.or-tools",
+  "name": "OR Tools",
+  "description": "Google's Operations Research tools FlatZinc interface",
+  "version": "${ortools_VERSION_MAJOR}.${ortools_VERSION_MINOR}.${ortools_VERSION_PATCH}",
+  "mznlib": "../ortools",
+  "executable": "${FZ_REL_INSTALL_BINARY}",
+  "tags": ["ortools", "cp", "lcg", "float", "int"],
+  "stdFlags": ["-a","-f","-n","-p","-r","-s","-v"],
+  "extraFlags": [
+    ["--cp_diffn_use_cumulative", "Add redundant cumulative constraint for Diffn", "bool:false:true", "true"],
+    ["--cp_disable_solve", "Force failure at the beginning of a search", "bool", "false"],
+    ["--cp_max_edge_finder_size", "Maximal number of tasks to post the edge finder in cumulative", "int", "50"],
+    ["--cp_model_check_intermediate_solutions", "All intermediate solutions found by the solver will be checked", "bool", "false"],
+    ["--cp_model_params", "Provide parameters interpreted as a text SatParameters proto", "string", ""],
```

Comment:
keep cp_model_params :-)

Reply:
As you can see, most flags come from the old cp solver. Peek cp_model_params, fz_int_as_floats, replace --fz_logging by -l.

Sorry for the time wasted.

Thanks again.

Reply:
No worries, it is was mostly some copy and pasting work during a meeting. It's good to know that some of the flags are from the old cp solver. I've pushed a updated version with just the flags as you suggested.